maps/src/main/java/com/gluonhq/impl/maps/TileImageView.java

```diff
@@ -29,6 +29,7 @@
 
 import com.gluonhq.maps.tile.TileRetriever;
 import com.gluonhq.maps.tile.TileRetrieverProvider;
+import javafx.application.Platform;
```

Comment:
Looks good, can you modify the header copyright to:
```
Copyright (c) 2020, 2022, Gluon
```

(and don't force-push, we'll squash before merging)